src/ActionsMenu.scss

```diff
@@ -0,0 +1,33 @@
+.ActionsMenu__header {
+    position:relative;
+    background-color: #cecccc;
+    margin-right: 0;
+    padding-bottom: 0.05rem;
+    padding-top: 0.05rem;
+    text-align:center;
+}
+
+.ActionsMenu__header-heading {
```

Comment:
Specify which grid column the heading is using and position the heading to centre by give equal amount of spaces using margin auto on top and bottom

`
.ActionsMenu__header-heading {
    font-size: inherit;
    font-weight: bold;
    margin: auto 0;
    grid-column-start: 2;
    grid-column-end: 3;
}
`